frontend/express/public/javascripts/countly/countly.helpers.js

```diff
@@ -12,13 +12,28 @@
 (function(CountlyHelpers) {
 
     CountlyHelpers.isPluginEnabled = function(name) {
-        if (countlyGlobal && countlyGlobal.pluginsFull && Array.isArray(countlyGlobal.pluginsFull) && countlyGlobal.pluginsFull.indexOf(name) > -1) {
-            if (countlyGlobal.plugins && Array.isArray(countlyGlobal.plugins) && countlyGlobal.plugins.indexOf(name) !== -1) {
-                return true;
+        if (countlyGlobal && countlyGlobal.pluginsFull && Array.isArray(countlyGlobal.pluginsFull)) {
```

Comment:
Please document these changes**
This function is missing jsdoc, can we add it ?